packages/account/src/Components/poa/file-uploader-container/file-uploader-container.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import { Icon, Text } from '@deriv/components';
 import { PlatformContext, isDesktop, WS } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
-import FileUploader from './file-uploader.jsx';
+import FileUploader from '../../file-uploader';
```

Comment:
Can this be changed to absolute path inside the project like `Components/file-uploader`

Reply:
Sure, will check it, thanks